### PR TITLE
Add `getExtensionConsoleUrl` utility

### PR DESCRIPTION
**Pull request by fregante**
## What does this PR do?

- Small change for https://github.com/pixiebrix/pixiebrix-extension/pull/6780

@twschiller  I was looking into the issue you were having, so I fixed this across the extension to avoid issues and improve consistency.

## Checklist

- [x] Add tests
- [x] New files added to `src/tsconfig.strictNullChecks.json` (if possible)
- [x] Designate a primary reviewer: @twschiller

### CODE REVIEW

File: src/sidebar/LoginPanel.tsx [hunk 1]
```diff
@@ -42,26 +43,23 @@ const DefaultLogin: React.FunctionComponent = () => (
...
-        href={extensionUrl}
-        target="_blank"
-        variant="primary"
-      >
-        <FontAwesomeIcon icon={faSignInAlt} /> Connect Account
-      </Button>
-    </Col>
-  );
-};
+const PartnerAuth: React.FunctionComponent = () => (
```
Comment (fregante) [inline src/sidebar/LoginPanel.tsx:46]: Whitespace change + inlined `getExtensionConsoleUrl()` call

**Comment by twschiller** [comment]
Thanks, @fregante. I had ChatGPT write a lint rule to enforce usage of the utility method

**Review by twschiller — APPROVED** [review]

**Comment by fregante** [comment]
There's already a no-forbiddden-syntax rule for this

**Comment by twschiller** [comment]
> There's already a no-forbiddden-syntax rule for this

TIL: sure, feel free to rewrite using that rule's configuration

**Pull request merged**